File: cielo24_.NET/WrapperCielo24/JSON/Task.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WrapperCielo24.JSON
{
    public class Task : JsonBase
    {
        [JsonProperty("TaskId")]
        public Guid TaskId { get; set; }
        [JsonProperty("TaskType")]
        public TaskType TaskType { get; set; }
        [JsonProperty("TaskRequestTime")]
        public DateTime TaskRequestTime { get; set; }
        [JsonProperty("TaskCompletionTime")]
        public DateTime TaskCompletionTime { get; set; }
        [JsonProperty("TaskInfo")]
        public string TaskInfo { get; set; }
        [JsonProperty("TaskStatus")]
        public TaskStatus TaskStatus { get; set; }
    }
}

File: cielo24_Java/cielo24/src/cielo24/json/Task.java
package cielo24.json;

import cielo24.utils.Guid;
import static cielo24.Enums.*;

import java.util.Date;

import com.google.gson.annotations.SerializedName;

public class Task extends JsonBase {
	@SerializedName("TaskId")
	public Guid taskId;
	@SerializedName("TaskType")
	public TaskType taskType;
	@SerializedName("TaskRequestTime")
	public Date taskRequestTime;
	@SerializedName("TaskCompletionTime")
	public Date taskCompletionTime;
	@SerializedName("TaskInfo")
	public String taskInfo;
	@SerializedName("TaskStatus")
	public TaskStatus taskStatus;
}